jobs/tests/test_jobs_view.py

```diff
@@ -30,6 +30,7 @@ def test_unauthenticated_user_enter_register(client):
 
 
 # Should move move another file - Good for Demo 3
+# Fix the typo here - issue #71
```

Comment:
There is already an issue open for this, there is no need to add a comment as well especially on  non related PR.